From c940ddd7aac80d1eba9f4b98b17ad37077f93632 Mon Sep 17 00:00:00 2001
From: Michal Hrusecky <michal.hrusecky@turris.com>
Date: Thu, 21 Nov 2024 13:04:55 +0100
Subject: [PATCH] target/linux: Add missing kernel options

When compiling Omnia kernel, there were few more options that needed
some decision. Those are not related to Omnia hardware but rather
related to the generic kernel update. Those options needed some
decision, so I'm making it.
---
 target/linux/generic/config-6.12 | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/target/linux/generic/config-6.12 b/target/linux/generic/config-6.12
index 8552f4f7d1..3b68d08cd1 100644
--- a/target/linux/generic/config-6.12
+++ b/target/linux/generic/config-6.12
@@ -1722,6 +1722,7 @@ CONFIG_DUMMY_CONSOLE_ROWS=25
 # CONFIG_DVB_B2C2_FLEXCOP_PCI is not set
 # CONFIG_DVB_B2C2_FLEXCOP_USB is not set
 # CONFIG_DVB_BCM3510 is not set
+# CONFIG_DVB_BUDGET_CORE is not set
 # CONFIG_DVB_CORE is not set
 # CONFIG_DVB_CX22700 is not set
 # CONFIG_DVB_CX22702 is not set
@@ -1742,6 +1743,7 @@ CONFIG_DUMMY_CONSOLE_ROWS=25
 # CONFIG_DVB_DIB7000P is not set
 # CONFIG_DVB_DIB8000 is not set
 # CONFIG_DVB_DIB9000 is not set
+# CONFIG_DVB_DM1105 is not set
 # CONFIG_DVB_DRX39XYJ is not set
 # CONFIG_DVB_DRXD is not set
 # CONFIG_DVB_DRXK is not set
@@ -1849,6 +1851,8 @@ CONFIG_DVB_MAX_ADAPTERS=16
 # CONFIG_DVB_ZL10036 is not set
 # CONFIG_DVB_ZL10039 is not set
 # CONFIG_DVB_ZL10353 is not set
+# CONFIG_MANTIS_CORE is not set
+# CONFIG_DVB_SMIPCIE is not set
 # CONFIG_DWC_PCIE_PMU is not set
 # CONFIG_DWC_XLGMAC is not set
 # CONFIG_DWMAC_DWC_QOS_ETH is not set
@@ -7417,8 +7421,10 @@ CONFIG_VHOST_MENU=y
 # CONFIG_VIDEO_CS3308 is not set
 # CONFIG_VIDEO_CS5345 is not set
 # CONFIG_VIDEO_CS53L32A is not set
+# CONFIG_VIDEO_CX18 is not set
 # CONFIG_VIDEO_CX231XX is not set
 # CONFIG_VIDEO_CX2341X is not set
+# CONFIG_VIDEO_CX23885 is not set
 # CONFIG_VIDEO_CX25821 is not set
 # CONFIG_VIDEO_CX25840 is not set
 # CONFIG_VIDEO_CX88 is not set
-- 
2.52.0

